recipes/ninja/1.10.x/conanfile.py

```diff
@@ -0,0 +1,52 @@
+import os
+import glob
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class NinjaConan(ConanFile):
+    name = "ninja"
+    description = "Ninja is a small build system with a focus on speed"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ninja-build/ninja"
+    settings = "os", "arch", "compiler"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            if self.settings.compiler.runtime != "MT":
+                raise ConanInvalidConfiguration("Only MT MSVC runtime is supported")
+
+    def _configure_cmake(self):
+        cmake = CMake(self, build_type="Release")
+        cmake.configure()
+        return cmake
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("ninja-%s" % self.version, self._source_subfolder)
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses",
+                  src=self._source_subfolder)
+        self.copy(pattern="ninja*", dst="bin", src="bin")
+
+        pdb_files = glob.glob(os.path.join(self.package_folder, 'bin', '*.pdb'), recursive=True)
+        for pdb in pdb_files:
+            os.unlink(pdb)
```

Comment:
```suggestion
            os.unlink(pdb)

    def package_id(self):
        del self.info.settings.compiler

```

Not required for installer package.